woocommerce/woocommerce-gutenberg-products-block
# Update Puppeteer E2E tests to WP 6.4.1 and disable failing ones

## What

Skips the E2E tests that are failing since the WC 8.3 release. Upgrades the WP Version used to 6.4.1

## Why

Because the WC 8.3 release only works with WP 6.3+ the current E2E setup (running on WP 6.2) doesn't work and tests don't run.

Upgrading to 6.3 is a problem because it causes lots of E2Es to fail.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. See E2E CI jobs pass.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Skipping.
